### PR TITLE
Realestate sold state: (1) User contact information

### PR DESCRIPTION
**Pull request by bstien**
# Why?
The user should enter their contact information so the agent can contact them.
This PR goes towards my other WIP branch.

# What?
- Added necessary views and view models.

# UI Changes
| Img | Gif |
| --- | --- |
| Screenshot 2021-11-10 at 12 55 23 | Simulator Screen Recording - iPhone 12 Pro - 2021-11-10 at 12 55 48 |

### CODE REVIEW

File: Sources/Components/RealestateSoldState/Subviews/UserContactInformationView.swift [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import UIKit
+import FinniversKit
+
+class UserContactInformationView: UIView {
+
+    // MARK: - Internal methods
```
Comment (ninarg) [inline Sources/Components/RealestateSoldState/Subviews/UserContactInformationView.swift:6]: internal properties I guess. well, the property is a method though 😆

File: Sources/Components/RealestateSoldState/RealestateSoldStateView.swift [hunk 2]
```diff
@@ -22,6 +22,28 @@ public class RealestateSoldStateView: UIView {
         return view
     }()
 
+    private lazy var userContactMethodView: UserContactInformationView = {
+        let view = UserContactInformationView(withAutoLayout: true)
+        view.configure(
+            with: "Hvordan kan vi kontakte deg?",
+            contactMethodModels: [
+                .init(
+                    identifier: "1",
```
Comment (ninarg) [inline Sources/Components/RealestateSoldState/RealestateSoldStateView.swift:31]: Will these strings be moved to the demo data or something? 😄 
the strings should be localized in ios-app

**Review by ninarg — APPROVED** [review]
Nice that you split your work into smaller PRs 🥇

File: Sources/Components/RealestateSoldState/RealestateSoldStateView.swift (hunk 2 above)
Comment (bstien) [reply Sources/Components/RealestateSoldState/RealestateSoldStateView.swift:31]: Yup, that's changed in the later PRs😌 Should probably have noted that in the description😄

**Pull request merged**